swerve_pico/main.c

```diff
@@ -0,0 +1,198 @@
+#include <stdio.h>
+
+#include "motor-library.cpp"
```

Comment:
same .h/.cpp comments from motor-library.cpp for all of these

Reply:
also, since all of your libraries are cpp code, your main file needs to be cpp as well, not c